modules/tensor_mechanics/test/tests/umat/orient_umat/tests

```diff
@@ -0,0 +1,62 @@
+[Tests]
+  # Hughes-Winget (hw) tests match Abaqus
+  [hw_reference]
+    type = 'Exodiff'
+    input = 'shear_top_umat.i'
+    exodiff = 'shear_top_umat_out.e'
+    library_mode = 'DYNAMIC'
+    valgrind = 'NONE'
+    cli_args = 'GlobalParams/decomposition_method=HughesWinget'
+    # skip test if test is being run out-of-tree. Issue Ref: #25279
+    installation_type = in_tree
+  []
+  [hw_reference_umat_output]
+    type = 'RunApp'
+    input = 'shear_top_umat.i'
+    library_mode = 'DYNAMIC'
+    valgrind = 'NONE'
+    cli_args = 'GlobalParams/decomposition_method=HughesWinget'
+    expect_out = 'STRESS_ 1   -0.1757425\n STRESS_ 2   -0.5954772\n STRESS_ 3   -0.0436915\n STRESS_ 4    0.9835696\n STRESS_ 5    0.0729212\n STRESS_ 6   -0.1727900\n '
+                 'STRAIN_ 1    0.0000101\n DSTRAIN_ 1    0.0000037\n STRAIN_ 2   -0.0004222\n DSTRAIN_ 2   -0.0001052\n STRAIN_ 3    0.0001507\n DSTRAIN_ 3    0.0000370\n '
+                 'STRAIN_ 4    0.0020475\n DSTRAIN_ 4    0.0005107\n STRAIN_ 5    0.0001519\n DSTRAIN_ 5    0.0000383\n STRAIN_ 6   -0.0003596\n DSTRAIN_ 6   -0.0000896\n '
+                 'COORDS_ 1    0.3915195\n COORDS_ 2    0.3857086\n COORDS_ 3    0.3873709'
+    # skip test if test is being run out-of-tree. Issue Ref: #25279
+    installation_type = in_tree
+  []
+
+  # rotated isotropic material module should give the same material (Cauchy) stress
+  [hw_rotated]
+    type = 'Exodiff'
+    input = 'shear_top_umat.i'
+    exodiff = 'shear_top_umat_out.e'
+    prereq = 'hw_reference'
+    library_mode = 'DYNAMIC'
+    valgrind = 'NONE'
+    cli_args = 'GlobalParams/decomposition_method=HughesWinget Materials/umat/orientation="-30 0 0"'
+  []
+  # but the stress computed in the intermediate reference frame should be different
+  # values match Abaqus
+  [hw_rotated_umat_output]
+    type = 'RunApp'
+    input = 'shear_top_umat.i'
+    library_mode = 'DYNAMIC'
+    valgrind = 'NONE'
```

Comment:
This test doesn't have in tree installation type.

Reply:
It's required for all UMAT or plugin tests as of now. That might change in the future.